ros/src/fsds_ros_bridge/src/airsim_ros_wrapper.cpp

```diff
@@ -131,9 +200,13 @@ void AirsimROSWrapper::create_ros_pubs_from_settings_json()
         set_nans_to_zeros_in_pose(*vehicle_setting);
 
         vehicle_name = curr_vehicle_name;
-        odom_pub = nh_.advertise<nav_msgs::Odometry>("odom", 10);
+        odom_pub = nh_.advertise<nav_msgs::Odometry>("for_testing/odom", 10);
         global_gps_pub = nh_.advertise<sensor_msgs::NavSatFix>("gps", 10);
         imu_pub = nh_.advertise<sensor_msgs::Imu>("imu", 10);
+        // TODO: remove track publisher at competition
+        track_pub = nh_.advertise<fsds_ros_bridge::Track>("for_testing/track", 10, true);
+        viz_track_pub = nh_.advertise<visualization_msgs::MarkerArray>("for_testing/viz/track", 1, true);

```

Comment:
undocumented

Reply:
what part? the viz marker?

Reply:
yes

Reply:
well if we are deleting it then don't bother ;)